openquake/server/settings.py

```diff
@@ -162,6 +166,27 @@
 
 FILE_UPLOAD_MAX_MEMORY_SIZE = 1
 
+# OpenQuake Standalone tools (IPT, Taxtweb, Taxonomy Glossary)
+STANDALONE_APPS = ()
+if STANDALONE:
+    INSTALLED_APPS += (
+        'openquakeplatform',
+    )
+
+    STANDALONE_APPS += (
+        'openquakeplatform_ipt',
+        'openquakeplatform_taxtweb',
+        'openquakeplatform_taxonomy',
+    )
+
+    INSTALLED_APPS += STANDALONE_APPS
+
+    FILE_PATH_FIELD_DIRECTORY = os.path.join(os.path.expanduser('~'), 'oqdata')
```

Comment:
This should be probably taken form the `openquake.cfg`

Reply:
The right function to call is `openquake.baselib.datastore.get_datadir()`